cvat-ui/src/utils/opencv-wrapper/opencv-interfaces.ts

```diff
@@ -1,8 +1,27 @@
-// Copyright (C) 2021 Intel Corporation
+// Copyright (C) 2021-2022 Intel Corporation
 //
 // SPDX-License-Identifier: MIT
 
 export interface ImageProcessing {
     processImage: (src: ImageData, frameNumber: number) => ImageData;
-    currentProcessedImage: number|undefined
+    currentProcessedImage: number | undefined;
+}
+
+export interface TrackingResult{
+    updated: boolean;
+    points: any[];
+}
+
+export interface TrackerModel {
+    name: string;
+    init: (src: ImageData, points: number[]) => void;
+    reinit: (points: number[]) => void;
+    update: (src: ImageData) => TrackingResult;
+}
+
+export interface OpenCVTracker{
+    name: string,
```

Comment:
So we have name in two structures: `OpenCVTracker` and `TrackerModel`. In first one we need it to render list of trackers without creating model instances. And in second one we use it to group `TrackedShape`s, `TrackedShape` has access only to `TrackerModel` which is in our case instance of `TrackerMILImplementation` class. For grouping reasons we can use class name but we need to render info message with tracker name, so that's why tracker name also used here.